app/cni/cmd/install/validation.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"github.com/pkg/errors"
+)
+
+func isValidConfFile(file string) bool {
+	parsed, err := parseFileToHashMap(file)
+	if err != nil {
+		log.Error(err, "could not unmarshal config file")
+		return false
```

Comment:
imho it's a bit awkward to log error and return false here. I would change return signature to `error` or `(bool, error)` and decide what to do with error on higher level